modules/tinymce/src/plugins/link/main/ts/core/Actions.ts

```diff
@@ -78,6 +82,10 @@ const createLink = (editor: Editor, selectedElm: Element, text: Optional<string>
     text.fold(
       () => {
         editor.execCommand('mceInsertLink', false, linkAttrs);
+        // Now the newly inserted link is selected. Move the cursor behind the new link, so the user can go on typing.
+        const rng = dom.createRng();
+        rng.setStartAfter(editor.selection.getEnd());
+        editor.selection.setRng(rng);
```

Comment:
I don't think the "more robust implementation" is desirable.

I'm not sure if the concern about `editor.selection.getEnd()` returning empty is justified.

Reply:
It's a valid concern, there's no inherent guarantee that the `mceInsertLink` event will create the link requested, but it usually would.

However the code you added doesn't actually require a link (despite what the comment suggests). It just moves the cursor to the end of whatever was selected. This will _always_ succeed.

Reply:
@coderabbitai: ignore